charts/portworx/values.yaml

```diff
@@ -67,7 +67,26 @@ telemetry: false
 
 deployOnMaster:  false                # For POC only
 csi: false                            # Enable CSI
-aut: false                            # Enable AutoPilot
+
+
+autopilot: 
+  enabled: false                      # Enable AutoPilot
+  image: ""                           # Autopilot image
+  lockImage: false                    # Enables locking Autopilot to the given image. 
```

Comment:
This field seems to be deprecated. Do we need to add this now?
@zoxpx please comment.